search-app/docs/api.md

```diff
@@ -1973,19 +1971,33 @@ By concept id and revision id
 
     curl -i "%CMR-ENDPOINT%/concepts/:concept-id/:revision-id"
 
-Examples:
+Plain examples, with and without revision ids:
 
     curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1"
-    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1.iso"
-    curl -i -H 'Accept: application/xml' "%CMR-ENDPOINT%/concepts/G100000-PROV1"
-    curl -i -H 'Accept: application/metadata+xml' "%CMR-ENDPOINT%/concepts/G100000-PROV1"
-    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1.json"
     curl -i "%CMR-ENDPOINT%/concepts/C100000-PROV1/1"
-    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1/2.echo10"
     curl -i "%CMR-ENDPOINT%/concepts/V100000-PROV1"
     curl -i "%CMR-ENDPOINT%/concepts/V100000-PROV1/1"
     curl -i "%CMR-ENDPOINT%/concepts/V100000-PROV1/2"
 
+File extension examples:
+
+    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1.iso"
+    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1.json"
+    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1/2.echo10"
+    curl -i -H "Accept: version=1.9" \
```

Comment:
Done.

I've also switched to using `umm_json` instead of `umm-json`, since the version addition doesn't work with the dash.